### PR TITLE
Added sticky footer with social links

### PR DESCRIPTION
**Pull request by aaronmboyd**
image

### CODE REVIEW

File: src/components/layouts/MainLayout/SocialLinks.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    name: "Git",
+    alt: "Link to GitHub source code",
+    imageSource: "/assets/GitHub-Mark/PNG/GitHub-Mark-Light-120px-plus.png",
+    link: "https://github.com/Celo-Fi",
+  },
+  {
+    name: "Discord",
+    alt: "Link to Discord discussion forum",
+    imageSource: "/assets/Discord-Logo-White.png",
+    link: "https://t.co/rqKBCVlh6q?amp=1",
```
Comment (BriungRi) [inline src/components/layouts/MainLayout/SocialLinks.tsx:25]: What link is this?

File: src/components/layouts/MainLayout/SocialLinks.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  {
+    name: "Telegram",
+    alt: "Link to Telegram discussion channel",
+    imageSource: "/assets/telegram-512.png",
+    link: "https://t.co/Be7yxqo0mG?amp=1",
```
Comment (BriungRi) [inline src/components/layouts/MainLayout/SocialLinks.tsx:31]: Ditto. Use the plain link please.

File: src/components/layouts/MainLayout/index.tsx [hunk 3]
```diff
@@ -14,12 +14,26 @@ export const MainLayout: React.FC<Props> = ({
   title = "Romulus",
 }: Props) => {
   return (
-    <Box sx={{ width: "100%", px: [4, 4], py: [3, 3] }}>
+    <Box
```
Comment (BriungRi) [inline src/components/layouts/MainLayout/index.tsx:17]: nit: Can use `Flex` here and remove `display: "flex"`

**Review by aaronmboyd — COMMENTED** [review]
Updated

**Pull request merged**